README.md

```diff
@@ -4,6 +4,77 @@ JS Client library to interact with Supabase Storage.
 
 - Documentation: https://supabase.io/docs/reference/javascript/storage-createbucket
 
+## Quick Start Guide
+
+### Installing the module
+
+``` bash
+npm install @supabase/storage-js
+```
+
+### Connecting to the storage backend
+
+``` bash
+# .env
+SUPABASE_STORAGE_URL=https://<project_ref>.supabase.co/storage/v1
+SUPABASE_SERVICE_KEY=<service_role>
+```
+
+``` js
+// index.hs
+import { SupabaseStorageClient } from '@supabase/storage-js'
+import 'dotenv/config'
+
+const STORAGE_URL = process.env.SUPABASE_STORAGE_URL;
+const SERVICE_KEY = process.env.SUPABASE_SERVICE_KEY; //! service key, not anon key
+
+const storage_client = new SupabaseStorageClient(
+    STORAGE_URL,
+    {
+        apikey: SERVICE_KEY,
+        Authorization: `Bearer ${SERVICE_KEY}`,
+    }
+);
+```
+
+### Handling resources
+
+- Create a new Storage bucket
+
+    ``` js
+    const { data, error } = await storage_client.createBucket("test_bucket", { public: false });
+    ```
+
+- Retrieve the details of an existing Storage bucke
+
+    ``` js
+    const { data, error } = await storage_client.getBucket("test_bucket");
+    ```
+
+- Updates a new Storage bucket
+
+    ``` js
+    const { data, error } = await storage_client.createBucket("test_bucket", { public: true });
+    ```
+
+- Remove all objects inside a single bucket
```

Comment:
The operation in the example doesn't match.

Reply:
Fixed in [this commit](https://github.com/supabase/storage-js/pull/47/commits/d6cf0313a897b738be9d292dcc349571cf81aec0)